src/EnergyPlus/HeatBalanceSurfaceManager.cc

```diff
@@ -8264,7 +8267,8 @@ void TestSurfTempCalcHeatBalanceInsideSurf(EnergyPlusData &state, Real64 TH12, S
                     } else {
                         ShowContinueError(state, format("...Internal Heat Gain (no floor) [{:.3R}] W", zone.InternalHeatGains / zone.FloorArea));
                     }
-                    if (state.dataAirflowNetwork->SimulateAirflowNetwork <= AirflowNetwork::AirflowNetworkControlSimple) {
+                    if (static_cast<int>(state.dataAirflowNetwork->SimulateAirflowNetwork) <=
+                        static_cast<int>(AirflowNetwork::AirflowNetworkControl::Simple)) {
                         ShowContinueError(state, format("...Infiltration/Ventilation [{:.3R}] m3/s", zone.NominalInfilVent));
```

Comment:
So ... IMO rather than do range comparisons on enumerations, it's clearer to do combined equality tests using the BITF and BITF_TEST_ANY macros. That way you can see at the call site which options are being compared rather than having to remember which options are <= AFNControl::Simple.

Reply:
I see. I'll figure out this with using `BITF`. Thank you for your comment.